extensions/wikia/AdEngine/js/run/desktop.run.js

```diff
@@ -77,6 +80,14 @@ require([
 		win.adslots2 = win.adslots2 || [];
 		adEngine.run(adConfigDesktop, win.adslots2, 'queue.desktop');
 
+		if (context.opts.sourcePoint && win.ads) {
```

Comment:
What do you think about not executing `adEngine` for regular slots if we detect ad blockers?

Nevermind. I don't think it's a best idea to risk executing our regular queue if SP detection from unknown reasons stops working ;)